packages/core/src/types/asyncapi.ts

```diff
@@ -417,23 +389,24 @@ const Schema: NodeType = {
     default: null,
     readOnly: { type: 'boolean' },
     writeOnly: { type: 'boolean' },
-    // xml: 'Xml',
     examples: { type: 'array' },
     example: { isExample: true },
     deprecated: { type: 'boolean' },
     const: null,
     $comment: { type: 'string' },
-    dependencies: { type: 'object' }, // TODO
+    additionalItems: (value: any) => {
+      return typeof value === 'boolean' ? { type: 'boolean' } : 'Schema';
+    },
+    dependencies: (value: any) => {
+      return Array.isArray(value) ? 'PropertyDependency' : 'Schema';
+    },
```

Comment:
The `dependencies` property couldn't be an array. It must be an object with additionalProperties which could be either an array or a schema: 
https://json-schema.org/draft-07/json-schema-validation#rfc.section.6.5.7

Could you update that to something like the following?

```suggestion
    dependencies: {
      properties: {}
      additionalProperties: value => Array.isArray(value) ? { type: array, ... } : 'Schema'
    },
```